lib/pages/add_bridge/add_bridge.dart

```diff
@@ -245,9 +371,15 @@ class BotController extends State<AddBridge> {
       }
     });
 
-    await Future.wait(socialNetworks
-        .where((network) => network.available)
-        .map((network) => pingSocialNetwork(network)));
+    await Future.wait(socialNetworks.where((network) => network.available).map((network) {
+      if (network.name == "Facebook Messenger") {
```

Comment:
`network.name != "Facebook Messenger"` should be more modular. I suggest adding a `supportsBridgev2Api: true` in `social_network.dart` and checking that.